realm/realm-library/src/main/java/io/realm/MutableRealmInteger.java

```diff
@@ -22,72 +22,69 @@
 
 
 /**
- * A MutableRealmInteger is a mutable, {@link java.lang.Long}-like numeric quantity.
- * It behaves almost exactly as a reference to a {@link java.lang.Long}. More specifically:
+ * A {@code MutableRealmInteger} is a mutable, {@link Long}-like numeric quantity.
+ * It behaves almost exactly as a reference to a {@link Long}. More specifically:
  * <ul>
- * <li>A MutableRealmInteger may have the value {@code null}.</li>
- * <li>The {@code .equals} operator compares the contained Long values. null-valued MutableRealmInteger are {@code .equals}</li>
- * <li>The {@code .compareTo} operator compares the contained Long values.  It considers {@code null} &lt; any value.</li>
- * <li>The {@code .increment} and {@code .decrement} operators throw {@code IllegalStateException} when applied to a null-valued MutableRealmInteger.</li>
+ * <li>A {@code MutableRealmInteger} may have the value {@code null}.</li>
```

Comment:
That's a fair point.  I didn't do it because I am not specifically referring to any particular implementation of the `.equals` method.  I don't think the link is useful.  But I might be wrong.  Will fix.